storage/internal/benchmarks/main.go

```diff
@@ -80,6 +80,8 @@ type benchmarkOptions struct {
 	numClients             int
 	workload               int
 	numObjectsPerDirectory int
+
+	gcsFuse bool
```

Comment:
we can rename this boolean flag: gcsFuse -> useGCSFuseConfig

Reply:
sure